sdk/go/common/resource/plugin/langruntime_plugin.go

```diff
@@ -236,3 +239,52 @@ func (h *langhost) Close() error {
 	}
 	return nil
 }
+
+func (h *langhost) InstallDependencies(directory string) error {
+	logging.V(7).Infof("langhost[%v].InstallDependencies(directory=%s) executing",
+		h.runtime, directory)
+	resp, err := h.client.InstallDependencies(h.ctx.Request(), &pulumirpc.InstallDependenciesRequest{
+		Directory:  directory,
+		IsTerminal: cmdutil.GetGlobalColorization() != colors.Never,
+	})
+
+	if err != nil {
+		rpcError := rpcerror.Convert(err)
+		logging.V(7).Infof("langhost[%v].InstallDependencies(directory=%s) failed: err=%v",
+			h.runtime, directory, rpcError)
+
+		// It's possible this is just an older language host, prior to the emergence of the InstallDependencies
+		// method.  In such cases, we will silently error (with the above log left behind).
+		if rpcError.Code() == codes.Unimplemented {
+			return nil
+		}
+
+		return rpcError
+	}
+
+	for {
+		output, err := resp.Recv()
+		if err != nil {
+			if err == io.EOF {
+				break
+			}
+			rpcError := rpcerror.Convert(err)
+			logging.V(7).Infof("langhost[%v].InstallDependencies(directory=%s) failed: err=%v",
+				h.runtime, directory, rpcError)
+			return rpcError
+		}
+
+		if len(output.Stdout) != 0 {
+			os.Stdout.Write(output.Stdout)
```

Comment:
Just sanity checking, langhost writes to stdout/stderr, or needs to "write" via gRPC?

Reply:
Yeh we use to call a subprocess and hook it's stdout to our stdout. Now we stream back stdout via grpc but we still want it to just write to stdout here.